package/libyui-rest-api.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Wed Oct 21 14:50:03 UTC 2020 - Rodion Iafarov <riafarov@suse.com>
+
+- Do not set json value to nullptr when YCheckBoxState is YCheckBox_dont_care
```

Comment:
NP: I would try to not exceed the width of the "----" line (e.g. by adding a line break after "YCheckBoxState is").

Reply:
Fixed.